apps/webhooks.md

```diff
@@ -0,0 +1,90 @@
+---
+layout: page
+key: apps-webhooks
+title: Webhooks
+---
+
+Webhooks allow developers to build apps that receive near-real-time information about events that happen in ePages shops.
+Instead of making constant API calls to check for specific events on the shop, you can register webhooks.
+So, directly after having registered for our [Developer Program](/#modal-popup), you might want to know more about how we have implemented webhooks.
+
+In order to receive the respective information about the events that your app subscribes to, webhooks require a callback URL.
+When the event happens, we'll send a `POST` request to your callback URL.
+Your app can then process the information send by us based on that event.
+
+Your app might require the following information:
+
+* an order has been created
+* an order has been updated
+* an order has been deleted
+* your app has been uninstalled.
+
+## Before you can get going
+
+Before you can send any requests or receive any responses, this is what you'll need:
+
+* Test shop: you can get a test shop by [joining the ePages Developer Program](/#modal-popup)
+* Client ID and Client Secret: received during [app creation](page:apps-create#get-your-credentials)
+* Access token: received during [app installation](page:apps-install)
+* SSL setup.
+
+## Ready to start creating a webhook
+
+To create a webhook, make a [`POST` request](page:apps-api-post-shopid-webhooks-information) to the respective webhook resource, including:
+
+* The app's `callbackUrl`
+* The `eventType`.
+
+By default, new webhooks will be set to **active**.
+If you want a webhook to be inactive initially, pass the following key-value `"active": false` with the request.
+
+An `HTTP 201` response indicates that the webhook was successfully set.
+
+## Webhook event types
```

Comment:
instead of creation a separate section for "webhook event types", just add two additional subsections to "Ready to start creating a webhook":
- the callback url (mention here that it has to be https)
- the event types (list the event types here)